### ISSUE TITLE
Add icon/label PM Cares Fund to fight Covid-19

### ISSUE THREAD

**Issue by helios1101** [covid19india/covid19india.github.io#672]
**Is your feature request related to a problem? Please describe.**
Since multiple people are visiting your  site, so a button to redirect to payment link on the page for donation towards PM cares Fund would help to gather more contribution and help the needy. 

**Describe the solution you'd like**
A button with a label "Play your Part", will redirect towards PM Cares Fund link, which will be handles by gateway afterwards. 

**Describe alternatives you've considered**
No such alternatives.

**Additional context**
Should we added in wiki, as why to contribute more and more, with some pics of people who lost their jobs in this time.

**Comment by smittal1209** [comment]
@shuklaayush Hi Ayush, I see this issue as somewhat related to the issue raised by me but the scope seems to be wider.

- I have added the scope to show all the national and state government relief funds details along with official sites link so that it will be easier for people to authenticate the official source before payment since that is a major concern for most people.

- I have also added the space to add NGOs that are directly working on targeted areas such as providing food and other essentials to the needy.

- Direct Payment pages are also there to each donation section and we can add multiple payment options like official online payment link, or pay via paytm or pay via google pay that directly opens the app or webpage to make direct payment.

- As mentioned in the future scopes of the issue, we can add Twitter and Instagram posts of official handles and celebrities or anyone who is playing their part to support the nation which can motivate people to come forward and donate as per their capabilities.

P.S. The issue is worth considering for providing a helping hand to the government and other people who are giving their best to help the needy.

**Comment by smittal1209** [comment]
@kant @tuxdna @tecoholic Any update on this team?

**Comment by helios1101** [comment]
Hey, @smittal1209 
I was wondering we can integrate all payment links of every state and show them when there is hover on map on state.
Additionally, we can add some page, illustrating authenticity of links and how each contributions will cause impact. Total number of clicks on each can be pictured, as total impact we have caused in this contributions.

**Comment by smittal1209** [comment]
@helios1101 I nice thought but lets break your concerns :

- The maps on the home page are showing the data and a lot of details already. So as per me, showing donate also there will mess up a lot of information on one place. Moreover we don't know that we will be able to provide donate links for all the states or not.

- Please check my PR on link [Here](https://github.com/covid19india/covid19india-react/issues/719) I have provided screenshots on how a simple look can be given.

- I have mentioned that we can add authenticated source of the links.

- Total Number of Clicks to show the impact is a very nice idea.

@kant @tuxdna @tecoholic Authors if you can have a look?

**Comment by srikanthlogic** [comment]
> Hey, @smittal1209
> I was wondering we can integrate all payment links of every state and show them when there is hover on map on state.
> Additionally, we can add some page, illustrating authenticity of links and how each contributions will cause impact. Total number of clicks on each can be pictured, as total impact we have caused in this contributions.

In case you are proceeding with this approach - You may find this useful - [State of payment digitization in Disaster Relief Funds -- India](https://docs.google.com/spreadsheets/d/1gTrGvX6HfGfd4zkfm7ctMgUBNwo6DhNUNvz5n5KVbw4/edit#gid=0)

**Comment by helios1101** [comment]
Sure, @smittal1209 
It looks much clean. I can start working on the same, as a new page.
Thanks for suggestion @srikanthlogic 
Will try to use the authentic links for the same.

**Comment by smittal1209** [comment]
@srikanthlogic Thanks for providing the data source.
Please refer to below screen shots on how the page will look like. I have already done the code and PR was raised but due to community guidelines was advised to first raise an issue for the same.
So, If this issue is approved, we can make other relevant changes and the PR can be merged.
Please approve the issue.
<img width="1391" alt="Screenshot 2020-04-01 at 6 21 39 PM" src="https://user-images.githubusercontent.com/23252949/78859349-adc9ad00-7a4c-11ea-88ca-3d0e8c976aba.png">

@helios1101 We work on the backend for the same once this is approved.

**Comment by helios1101** [comment]
Sure, @smittal1209.
We can work together around.

**Comment by nrishabh** [comment]
The page at http://bit.ly/HelpIndiaFightCovid19 (Thanks to Amnesty International) might help in gathering data about various fundraisers.

**Comment by sudevschiz** [comment]
This could be a potential section in the "Essential" page that now have. 

@nikita-11 @sama007 thoughts?

**Comment by sama007** [comment]
Yes. It can be put in the essentials page

**Comment by smittal1209** [comment]
@sama007 @sudevschiz Agreed that it can be integrated with the essentials page but lets focus on some points :

- Segregation of Responsibility : Essentials page is to provide people the information that can help them in any way that's why its being called Essentials.

- Data on the Donations page will increase in future and thus I would prefer it to be a separate page clearly visible from the home page.

- If we put it under Essentials Page, People who are seeking the reports and are willing to donate might miss them because Essentials can never meant to be Donations.

Please consider the points and close this issue if possible to be ASAP because situations are getting worsened.
@helios1101 Your point on this?

**Comment by nikita-11** [comment]
> This could be a potential section in the "Essential" page that now have.
> 
> @nikita-11 @sama007 thoughts?

It can be put there. But I don't understand the purpose of highlighting the PM Cares fund as such! If we want 

> This could be a potential section in the "Essential" page that now have.
> 
> @nikita-11 @sama007 thoughts?

I think the essentials page Shouldn't include the Fundraisers. Though currently it includes this as a category. But going for a separate donations page makes complete sense.

The data team has done commendable work in gathering so many on ground NGOs, which have their official sites as well as valid social media handles and are risking their lives to provide basic amenities to poor. So in my view we shouldn't highlight the centre or CM funds as such. But provide equal weightage to all these donation links and attract audience there as a whole. Data is already as a Fundraisers category in Essential sheets. And it makes all the more sense to separate that category from essentials as the name clearly suggests! We do need a New Donate Asap!

**Comment by smittal1209** [comment]
@nikita-11 A great example given.
You are correct in the sense of not showing the PM Cares Fund and Other State Govt. Funds on the page but that section is included in equal weightage to other NGOs fundraisers. (Refer the Screenshot below)
My only concern to show these on the donations page is due to following points : 

- Provide people a sense of authenticity that they are directly contributing to National and State Govt. funds and the Govt. would be able to use those funds better than anyone else.

- Just a choice to contribute to any of the Govt. or NGOs Funds.

- We would not like to be partial in providing the information to raise funds to people. Our concern is to provide them authenticate sources of all the Available Channels (be it Govt. or non Govt.) by which contributions can be made. 

- It finally should be decision of the contributor to choose from which channel he likes to donate.

<img width="1391" alt="Screenshot 2020-04-01 at 6 21 39 PM" src="https://user-images.githubusercontent.com/23252949/79075378-1bf1c680-7d10-11ea-8718-53672ed7a2f2.png">

**Comment by sudevschiz** [comment]
The links that we sponsor need to be well thought out. Now that the economy is extremely volatile, requesting money has lot of issues behind it. 
Putting this on hold.
